libs/portal-framework-ui/src/components/data-table/DefaultTableLayout.tsx

```
@@ -0,0 +1,132 @@
+  
+  return (
+    <div className={className}>
+      {header && <div className="mb-4">{header}</div>}
+      <div className="border rounded-md">
+        <UITable>
+          <TableHeader>
+            {table.getHeaderGroups().map((headerGroup) => (
+              <TableRow key={headerGroup.id}>
+                {headerGroup.headers.map((header) => {
+                  // Use the shared hook for mobile column hiding
+                  const isHidden = useMobileColumnHiding({
```

Biome (2.1.2)

[error] This hook is being called from a nested function, but all hooks must be called unconditionally from the top-level component.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)